src/main/java/org/opensearch/security/support/ConfigConstants.java

```diff
@@ -320,6 +320,15 @@ public enum RolesMappingResolution {
     public static final String TENANCY_GLOBAL_TENANT_NAME = "global";
     public static final String TENANCY_GLOBAL_TENANT_DEFAULT_NAME = "";
 
+    // On-behalf-of endpoints settings
+    public static final Integer OBO_DEFAULT_EXPIRY_SECONDS = 5 * 60;
+    public static final Integer OBO_MAX_EXPIRY_SECONDS = 10 * 60;
```

Comment:
These values are not loaded from the configuration so they shouldn't be in this file, move them to where they are used

Reply:
Ya I have moved them outside of here.